openfe/utils/ligand_utils.py

```diff
@@ -0,0 +1,23 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from gufe import LigandAtomMapping
```

Comment:
just curious, what is the reason for avoiding importing this during runtime?

Reply:
As I only use it for a type hint, I thought it might be best to skip the import. I'm happy to change it though!